lib/index.js

```diff
@@ -1,33 +1,33 @@
 #!/usr/bin/env node
 
-import path from 'path';
+import fs from 'fs';
 import commander from 'commander';
-import requireCwd from 'req-cwd';
+import resolveCwd from 'resolve-cwd';
 
 commander
   .usage('[options] <tasks runner> [arguments]')
   .option('-f, --file, <file>', 'tasks file path, tasks.js by default', 'tasks.js')
   .option('-p, --preset, <preset>', 'tasks preset')
   .parse(process.argv);
 
-let tasks = null;
+let modulePath = null;
 
 if (commander.preset) {
-  try {
-    tasks = requireCwd(commander.preset);
-  } catch (e) {
+  modulePath = resolveCwd(commander.preset);
+  if (!modulePath || !fs.existsSync(modulePath)) { // eslint-disable-line
     console.error(`Unable to find "${commander.preset}" preset, please check it again`);
     process.exit(1);
   }
 } else {
-  try {
-    tasks = require(path.resolve(commander.file));
-  } catch (e) {
+  modulePath = resolveCwd(`./${commander.file}`);
+  if (!modulePath || !fs.existsSync(modulePath)) { // eslint-disable-line
```

Comment:
trying to find a reason to have `!fs.existsSync(modulePath)` here and can't. looks like ``resolveCwd(`./${commander.file}`)`` already solves all the local path cases. part of my testing code:

```js
let modulePath = null;

if (commander.preset) {
  modulePath = resolveCwd(commander.preset);

  if (!modulePath) {
    console.error(`Unable to resolve "${commander.preset}" preset`);
    process.exit(1);
  }
} else {
  modulePath = resolveCwd(`./${commander.file}`);

  if (!modulePath) {
    console.error(`Unable to find "${commander.file}" file`);
    process.exit(1);
  }
}

const tasks = require(modulePath);
```

what do you think?